src/wallet/wallet.h

```diff
@@ -553,6 +553,12 @@ class CAccountingEntry
     std::vector<char> _ssExtra;
 };
 
+enum CreateTransactionFlags {
+    CREATE_TX_DEFAULT     = 0,
+    CREATE_TX_DONT_SIGN   = (1U << 0),
```

Comment:
Please invert this. Not only is it unintuitive to have it backward, it also contradicts the current function signature (that is, passing `true` or `false` will silently get the opposite behaviour).

```C++
    CREATE_TX_DEFAULT = 1,
    CREATE_TX_SIGN = 1,
```